libs/facilities/frontend/shell/src/lib/shell.routes.ts

```diff
@@ -23,12 +23,24 @@ export const FACILITIES_SHELL_ROUTES: Route[] = [
 			{
 				// Route to facilities with issues
 				path: 'issues',
-				redirectTo: 'not-found',
+				data: {
+					breadcrumbs: {
+						label: 'Issues',
+						url: 'facilities/issues',
+					},
+				},
+				loadComponent: () => import('facilities-frontend-view').then((m) => m.XdIssuesPage),
 			},
 			{
 				// Route to detail page
 				path: ':id',
-				redirectTo: 'not-found',
+				data: {
+					breadcrumbs: {
+						label: 'Details',
+						url: 'facilites/:id',
```

Comment:
the breadcrumbs with the details pages are not functional yet because I did not know how to get the id here. Right now I think it just sends you to /:id and not to the actual id

Reply:
You need to use an interceptor for that.

Reply:
We will look into it more as part of the detail Page Issue